core/modules/utils/dom/scroller.js

```diff
@@ -69,7 +69,13 @@ Handle a scroll event hitting the page document
 PageScroller.prototype.scrollIntoView = function(element,callback,options) {
 	var self = this,
 		duration = $tw.utils.hop(options,"animationDuration") ? parseInt(options.animationDuration) : $tw.utils.getAnimationDuration(),
-		srcWindow = element ? element.ownerDocument.defaultView : window;
+		srcWindow = element ? element.ownerDocument.defaultView : window,
+		closestElement = element.closest(".tc-scroll-container");
```

Comment:
I haven't had the opportunity to review the merits of the approach yet, but just a quick heads up that .closest() is not available in IE11 and cannot be used in the core. However I do not recommend putting more effort into this before you have had feedback.

Reply:
Thanks @saqimtiaz , yes I'm aware that .closest() is not available in IE11 ...
There are possible workarounds, which are simple enough but as you say, I won't put more effort into this before I have had feedback